central/sensor/service/service_impl.go

```diff
@@ -146,6 +171,11 @@ func (s *serviceImpl) Communicate(server central.SensorService_CommunicateServer
 		}
 	}
 
+	if svcType == storage.ServiceType_REGISTRANT_SERVICE {
+		// Terminate connection using a CRS certificate at this point.
```

Comment:
```suggestion
		// Terminate connection which uses a CRS certificate at this point.
```
"using" could be understood as if the CRS was the tool used to perform the termination 😅